src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -8,6 +8,11 @@ public class Bulldozer extends Machine {
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color) {
+        this.setName(name);
+        this.setColor(color);
```

Comment:
That was my solution But the abstract class Machine has name and color as private fields so that I can access them only through inherited functions - getters & setters. 
Is it ok to change the code already provided within a template?

Reply:
I deleted the custom constructor, as it is required by checkstyle to keep `name` and `color` fields as private ones. So I can't assign them directly in the subclass.